### PR TITLE
faq/lwjgl: make "regenerate your run configs" clearer

### PR DESCRIPTION
**Pull request by Leo40Git**
`regenerate your run configs` isn't exactly helpful by itself, is it?

### CODE REVIEW

**Comment by modmuss50** [comment]
This isnt correct either. As loom doesnt regen your run configs if they exist.

**Comment by Leo40Git** [comment]
Okay, *now* it should be clearer.

File: tags/faq/lwjgl.ytag [hunk 1]
```diff
@@ -4,4 +4,5 @@ type: text
 
 If you're getting `java.lang.UnsatisfiedLinkError: Failed to locate library: lwjgl.dll` when running your project:
   1. delete `<project dir>/.gradle/loom-cache/launch.cfg`
```
Comment (YTG1234) [inline tags/faq/lwjgl.ytag:6]:
```suggestion
  1. Delete `<project dir>/.gradle/loom-cache/launch.cfg`
```

File: tags/faq/lwjgl.ytag [hunk 2]
```diff
@@ -4,4 +4,5 @@ type: text
...
-  2. regenerate your run configs
+  2. delete your run configs
+  3. regenerate your run configs by reimporting the Gradle project
```
Comment (YTG1234) [inline tags/faq/lwjgl.ytag:8]:
```suggestion
  2. Delete your run configs from the configuration menu
  3. Regenerate your run configs by reimporting the Gradle project
```

File: tags/faq/lwjgl.ytag [hunk 3]
```diff
@@ -4,4 +4,5 @@ type: text
 
 If you're getting `java.lang.UnsatisfiedLinkError: Failed to locate library: lwjgl.dll` when running your project:
```
Comment (YTG1234) [inline tags/faq/lwjgl.ytag:5]:
```suggestion
If you're getting `java.lang.UnsatisfiedLinkError: Failed to locate library: lwjgl.dll` (might be `so` or `dylib` for different systems) when running your project:
```

**Review by YTG1234 — CHANGES_REQUESTED** [review]

File: tags/faq/lwjgl.ytag (hunk 1 above)
Comment (YTG1234) [reply tags/faq/lwjgl.ytag]: help how do I close this lol

Comment (Leo40Git) [reply tags/faq/lwjgl.ytag]: I already closed it by resolving it, don't worry about it :P

Comment (YTG1234) [reply tags/faq/lwjgl.ytag]: ah, apparently I had to refresh the page to see it being resolved

File: tags/faq/lwjgl.ytag [hunk 4]
```diff
@@ -2,6 +2,7 @@ type: text
 
 ---
 
-If you're getting `java.lang.UnsatisfiedLinkError: Failed to locate library: lwjgl.dll` when running your project:
-  1. delete `<project dir>/.gradle/loom-cache/launch.cfg`
-  2. regenerate your run configs
+If you're getting `java.lang.UnsatisfiedLinkError: Failed to locate library: lwjgl.dll/dylib/so` when running your project:
+  1. Delete `<project dir>/.gradle/loom-cache/launch.cfg`
```
Comment (modmuss50) [inline tags/faq/lwjgl.ytag:6]: This isn’t needed, it’s regenerated each time and likely already has the correct content. The issue is when moving a work space the path in the run config is wrong.

File: tags/faq/lwjgl.ytag [hunk 5]
```diff
@@ -2,6 +2,7 @@ type: text
...
+  2. Delete your run configs
```
Comment (modmuss50) [inline tags/faq/lwjgl.ytag:7]: Sadly this is different depending on the IDE, it’s more helpful than before though but still not super helpful.

File: tags/faq/lwjgl.ytag [hunk 6]
```diff
@@ -2,6 +2,7 @@ type: text
...
+  3. Regenerate your run configs by reimporting the Gradle project
```
Comment (modmuss50) [inline tags/faq/lwjgl.ytag:8]: I would say something along the lines of following the instructions you used to setup the workspace to begin with

**Review by modmuss50 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**